src/components/SeriesNav/SeriesNav.js

```diff
@@ -110,22 +121,26 @@ function SeriesNav({
                 series.title
               )}
             </InfoBoxTitle>
-            <InfoBoxText>
-              {series.description}
-              {titlePath &&
-                t.elements('styleguide/SeriesNav/seriesoverview/link', {
-                  link: (
-                    <Link href={titlePath} passHref>
-                      <Editorial.A>
-                        {t('styleguide/SeriesNav/seriesoverview')}
-                      </Editorial.A>
-                    </Link>
-                  )
-                })}
-            </InfoBoxText>
+            {showSeriesDescripion && (
+              <InfoBoxText>
+                {series.description}
+                {titlePath &&
+                  t.elements('styleguide/SeriesNav/seriesoverview/link', {
+                    link: (
+                      <Link href={titlePath} passHref>
+                        <Editorial.A>
+                          {t('styleguide/SeriesNav/seriesoverview')}
+                        </Editorial.A>
+                      </Link>
+                    )
+                  })}
+              </InfoBoxText>
+            )}
           </InfoBox>
           {inlineAfterDescription}
         </Center>
+      ) : showSeriesDescripion ? (
+        <Editorial.P {...styles.description}>{series.description}</Editorial.P>
```

Comment:
We should avoid setting [css rules on custom elements](https://styleguide.republik.ch/dev/colors#avoid-custom-elements).

```suggestion
        <Editorial.P style={{padding: '0px 15px'}}>{series.description}</Editorial.P>
```

or

```suggestion
        <div {...styles.description}>
          <Editorial.P>{series.description}</Editorial.P>
        </div>
```

It does currently work:
<img width="316" alt="Screenshot 2021-10-08 at 06 03 25" src="https://user-images.githubusercontent.com/410211/136496756-248cc931-9f9c-4713-9842-41db6190587e.png">

But if any other rule on the element started to set padding it would be random (execution order) dependant which padding would win.